staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -73,16 +91,50 @@ var (
 	  kubectl create secret generic my-secret --from-env-file=path/to/bar.env`))
 )
 
-// SecretGenericOpts holds the options for 'create secret' sub command
-type SecretGenericOpts struct {
-	CreateSubcommandOptions *CreateSubcommandOptions
+// SecretGenericOptions holds the options for 'create secret' sub command
+type SecretGenericOptions struct {
```

Comment:
`CreateSecretOptions` is the suggested name for this structure, since you're already touching it.